docs/autorender.md

```diff
@@ -99,6 +99,9 @@ in addition to two auto-render-specific keys:
 - `errorCallback`: A callback method returning a message and an error stack
   in case of an critical error during rendering. The default uses `console.error`.
 
+- `preProcess`: A callback function, `(math: string): string`, used to process
```

Comment:
I think `(math: string) => string` would be a bit clearer.

Reply:
Done.